src/main/java/io/odpf/firehose/sink/mongodb/util/MongoResponseHandler.java

```diff
@@ -0,0 +1,50 @@
+package io.odpf.firehose.sink.mongodb.util;
+
+import com.mongodb.BulkWriteError;
+import com.mongodb.BulkWriteException;
+import com.mongodb.bulk.BulkWriteResult;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.model.WriteModel;
+import io.odpf.firehose.metrics.Instrumentation;
+import lombok.experimental.UtilityClass;
+import org.bson.Document;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The type Mongo response handler.
+ */
+@UtilityClass
+public class MongoResponseHandler {
```

Comment:
Ok. what I wanted to do was this:

it's a normal class and not a util. 
It will have mongocollection and instrumentation as a member.  You can initialise this object inside MongoDBSink constructor. 
It will have public methods  Not static. 
* processRequest(List<> requests) {}
It will be called from mongosink.

Reply:
ok working on it

Reply:
fixed